### PR TITLE
Request booking feature

### LINKED ISSUE #9: User can make a booking request for a space

### PR DESCRIPTION
**Pull request by kerisic**
Completed request booking specification where user can view a space page, click on request booking button and be redirected to the requests summary page with details of the space.

### CODE REVIEW

File: lib/request.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Request
```
Comment (spaek14) [inline lib/request.rb:1]: love the implementation of a class for requests

**Review by spaek14 — APPROVED** [review]
looks amazing, really beautifully done!

**Review by olgachmaj — APPROVED** [review]
Really nice code

**Comment by AJ8GH** [comment]
Thanks for your comments :)

**Pull request merged**